cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -66,23 +66,15 @@ CREATE INDEX [index_name] [IF NOT EXISTS]
 FOR (n:LabelName)
```

Comment:
There is a table of content at the beginning of this file which still refers to replacing a composite index